### ISSUE TITLE
Virtualbox module failed after install xanmod kernel.

### ISSUE THREAD

**Issue by abcfy2** [xanmod/linux#40]
```
$ sudo vboxconfig 
vboxdrv.sh: Stopping VirtualBox services.
depmod: WARNING: could not open /lib/modules/5.4.24/modules.order: No such file or directory
depmod: WARNING: could not open /lib/modules/5.4.24/modules.builtin: No such file or directory
vboxdrv.sh: Starting VirtualBox services.
vboxdrv.sh: Building VirtualBox kernel modules.
vboxdrv.sh: failed: modprobe vboxdrv failed. Please use 'dmesg' to find out why.

There were problems setting up VirtualBox.  To re-start the set-up process, run
  /sbin/vboxconfig
as root.  If your system is using EFI Secure Boot you may need to sign the
kernel modules (vboxdrv, vboxnetflt, vboxnetadp, vboxpci) before you can load
them. Please see your Linux system's documentation for more information.
```

And I find the sulotion:

```
$ sudo rm -fr /lib/modules/5.4.24
$ sudo ln -s /lib/modules/5.4.24-xanmod13 /lib/modules/5.4.24
$ sudo vboxconfig
vboxdrv.sh: Stopping VirtualBox services.
vboxdrv.sh: Starting VirtualBox services.
vboxdrv.sh: Building VirtualBox kernel modules.
```

I hope this issue could be fixed. Thanks.

**Comment by abcfy2** [comment]
Maybe you can create this symbol link in `postinstall` script? I don't know the deb package workflow.

**Comment by Potterli20** [comment]
https://www.cnblogs.com/4geek/p/12187463.html
 永久解决Linux更新后Vmware无法打开提示“kernel module updater”
The article in Chinese

**Comment by figue** [comment]
> https://www.cnblogs.com/4geek/p/12187463.html
> 永久解决Linux更新后Vmware无法打开提示“kernel module updater”
> The article in Chinese

Thanks but this tip is for VMware.

Guys, we know that in the few days after a "major" kernel upgrade some modules as Nvidia, Virtualbox or VMware could fail. Please be patient until upstream fix the issue and if you can't, stay in 5.4.x branch. These issues are not related to Xanmod itself.

**Comment by xanmod** [comment]
Use the latest virtualbox 6.x w/ Linux 5.4 or later.
